match_score.py

```diff
@@ -5,19 +5,24 @@
 
 
 
-def compute_match_score(listOfGenomes,genomeCoverage, templates, templateKmers, filteringKmerLength, matchingKmerLength, f0):
+def compute_match_score(listOfGenomes, templates, templateKmers, filteringKmerLength, matchingKmerLength, f0):
     numberOfGenomes = len(listOfGenomes)
     matchScore = [[]] * numberOfGenomes
-
+    genomeCoverage=[]
     for num, genome in enumerate(listOfGenomes):
-
-        reads1 = list(SeqIO.parse(genome + "_1.fq", "fastq"))
-        reads2 = list(SeqIO.parse(genome + "_2.fq", "fastq"))
-
+        try:
+            reads1 = list(SeqIO.parse(genome + "1.fq", "fastq"))
+            reads2 = list(SeqIO.parse(genome + "2.fq", "fastq"))
+        except:
+            reads1 = list(SeqIO.parse(genome + "1.fastq", "fastq"))
+            reads2 = list(SeqIO.parse(genome + "2.fastq", "fastq"))
+
+        coverage = len(reads1)/1000.0 #assuming all genomes are roughly the same length - # of reads should be proportional to coverage
```

Comment:
I believe coverage is reads1*2, as reads1 only contains half the reads

Reply:
It's a completely arbitrary number proportional to the coverage of the genome. length of reads1 = length of reads2. This is just a way of not having to specify coverage for genomes with identical genome length (and identical read length!)